pyramid-app/{{ cookiecutter.slug }}/tests/functional/app_test.py

```diff
@@ -1,11 +1,21 @@
+import pytest
 from webtest import TestApp
 
 from {{ cookiecutter.package_name }}.app import create_app
 
 
-def test_it():
-    app = TestApp(create_app({}))
-
+def test_index(app):
     response = app.get("/")
 
-    assert response.json == {"Hello": "Pyramid!"}
+    assert response.text == "Hello world!"
```

Comment:
Just for variety I changed the default root endpoint from JSON to text how that we have a JSON `/_status` endpoint